### PR TITLE
Create messages.yml

### PR DESCRIPTION
**Pull request by ArresT-AnarchY**
Add Turkish translation

@gera2ld

### CODE REVIEW

File: src/_locales/tr/messages.yml [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+  message: Yükleniyor ...
+labelSettings:
+  description: Label shown on the top of settings page
+  message: Ayarlar
+labelAutoUpdate:
+  description: Option to allow automatically checking scripts for updates every 24 hours.
+  message: Scriptleri her gün otomatik olarak kontrol et
+labelIgnoreGrant:
+  description: Option to suppress no @grant warnings.
+  message: @grant bulunmama uyarılarını engelle
```
Comment (gera2ld) [inline src/_locales/tr/messages.yml]: Strings starting with `@` must be wrapped in quotes (`message: '...'`).

File: src/_locales/tr/messages.yml [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+  description: Check a script for updates.
+  message: Güncellemeleri denetle
+msgImported:
+  description: >-
+    Message shown after import. There is an argument referring to the count of
+    scripts imported.
+  message: $1 adet içeriye aktarıldı.
+hintUseDownloadURL:
+  description: Shown as a place holder for @updateURL when it is not assigned
+  message: @downloadURL Kullan
```
Comment (gera2ld) [inline src/_locales/tr/messages.yml]: Must be wrapped in quotes.

File: src/_locales/tr/messages.yml [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+  description: Message shown when custom CSS is saved.
+  message: Geçerli stil güncellendi.
+labelCustomCSS:
+  description: Label for custom CSS section.
+  message: Geçerli Stil
+descCustomCSS:
+  description: Geçerli stil bölüm açıklamaları
+  message: >-
+    Custom CSS for options page and script installation page. If you are not
+    sure what this is for, please do not edit it.
```
Comment (gera2ld) [inline src/_locales/tr/messages.yml]: `message` should be translated instead of `description`.

**Comment by gera2ld** [comment]
Thanks for your work and sorry for the bad experience to add translations. I will create a site to make it easier in the future.

**Comment by ArresT-AnarchY** [comment]
i have no node.js just now. i fix my false. what can i do now? send it on here? or pull request again?

ps. excuse i am newbie for github :(

**Comment by gera2ld** [comment]
@ArresT-AnarchY Just update it and push to the same branch again.

**Comment by ArresT-AnarchY** [comment]
please control and excuse my newbies these. :) @gera2ld

File: src/_locales/tr/messages.yml [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+  message: Yükleniyor ...
+labelSettings:
+  description: Label shown on the top of settings page
+  message: Ayarlar
+labelAutoUpdate:
+  description: Option to allow automatically checking scripts for updates every 24 hours.
+  message: Scriptleri her gün otomatik olarak kontrol et
+labelIgnoreGrant:
+  description: Option to suppress no @grant warnings.
+  message: `@grant` izin bulunmama uyarılarını engelle
```
Comment (gera2ld) [inline src/_locales/tr/messages.yml]: Still invalid, strings cannot start with `@` or `` ` `` in yaml, please wrap them in quotes. (`'...'`)

File: src/_locales/tr/messages.yml [hunk 5]
```diff
@@ -0,0 +1,420 @@
...
+  description: Check a script for updates.
+  message: Güncellemeleri denetle
+msgImported:
+  description: >-
+    Message shown after import. There is an argument referring to the count of
+    scripts imported.
+  message: $1 adet içeriye aktarıldı.
+hintUseDownloadURL:
+  description: Shown as a place holder for @updateURL when it is not assigned
+  message: `@downloadURL` Kullan
```
Comment (gera2ld) [inline src/_locales/tr/messages.yml]: Same problem here.

File: src/_locales/tr/messages.yml [hunk 6]
```diff
@@ -0,0 +1,420 @@
...
+  message: Yetkilendiriliyor
+msgSavedCustomCSS:
+  description: Message shown when custom CSS is saved.
+  message: Geçerli stil güncellendi.
+labelCustomCSS:
+  description: Label for custom CSS section.
+  message: Geçerli Stil
+descCustomCSS:
+  description: Description of custom CSS section.
+ message: >-
```
Comment (gera2ld) [inline src/_locales/tr/messages.yml]: Syntax error: lack of a space here.

File: src/_locales/tr/messages.yml [hunk 7]
```diff
@@ -0,0 +1,420 @@
...
+  message: Yükleniyor ...
+labelSettings:
+  description: Label shown on the top of settings page
+  message: Ayarlar
+labelAutoUpdate:
+  description: Option to allow automatically checking scripts for updates every 24 hours.
+  message: Scriptleri her gün otomatik olarak kontrol et
+labelIgnoreGrant:
+  description: Option to suppress no @grant warnings.
+  message: '@grant' izin bulunmama uyarılarını engelle
```
Comment (gera2ld) [inline src/_locales/tr/messages.yml]: It should be either
```
message: '@grant izin bulunmama uyarılarını engelle'
```
or
```
message: "'@grant' izin bulunmama uyarılarını engelle"
```

File: src/_locales/tr/messages.yml [hunk 8]
```diff
@@ -0,0 +1,420 @@
...
+  description: Check a script for updates.
+  message: Güncellemeleri denetle
+msgImported:
+  description: >-
+    Message shown after import. There is an argument referring to the count of
+    scripts imported.
+  message: $1 adet içeriye aktarıldı.
+hintUseDownloadURL:
+  description: Shown as a place holder for @updateURL when it is not assigned
+  message: '@downloadURL' Kullan
```
Comment (gera2ld) [inline src/_locales/tr/messages.yml]: Same problem here.

**Pull request merged**